ui/src/app/job-list/job-list-resolver.service.ts

```diff
@@ -43,8 +43,9 @@ export class JobListResolver implements Resolve<JobStream> {
     }
 
     const jobStream = new JobStream(this.jobManagerService, jobsRequest);
+    const numToLoad = settings.pageSize < initialBackendPageSize ? initialBackendPageSize : settings.pageSize * 2;
```

Comment:
I don't totally follow this logic, but I'd just take `max(initialBackendPageSize, settings.pageSize * 2)`